### PR TITLE
[NLU-1471] use tags to build package

### PR DESCRIPTION
**Pull request by sanhal**
Using tags to build package instead of branch

### CODE REVIEW

**Comment by sanhal** [comment]
Created tag, 
Investigated pushing mac version of the package, It is possible to push multiple wheels based on a platform for the same version in jfrog.
But I will not be able to push packages from the local system as we do not have to access to do it. 
We can look at other solutions if this becomes necessary in the future.

**Review by samhickey25 — APPROVED** [review]

File: python/ffi/Cargo.toml [hunk 1]
```diff
@@ -12,6 +12,6 @@ crate-type = ["cdylib"]
 failure = "0.1"
 libc = "0.2"
 ffi-utils = { git = "https://github.com/snipsco/snips-utils-rs", rev = "291ce1d" }
-snips-nlu-parsers-ffi-macros = { git = "https://github.com/purecloudlabs/snips-nlu-parsers-genesys.git", branch = "NLU-1383" }
+snips-nlu-parsers-ffi-macros = { git = "https://github.com/purecloudlabs/snips-nlu-parsers-genesys.git", tag = "snips-nlu-parsers-genesys-0.1.2" }
```
Comment (schutza) [inline python/ffi/Cargo.toml]: any reason to not just tag with version number? `0.1.2`

Comment (sanhal) [reply python/ffi/Cargo.toml]: we already have a tag `0.1.2` from the original repo ( fork maintains them )

Comment (schutza) [reply python/ffi/Cargo.toml]: ok, can we find a different / more succinct prefix then? `snips-nlu-parsers-genesys` just duplicates what we already know from the artefact. How about `0.1.2-genesys` or `genesys-0.1.2` ?

**Review by schutza — APPROVED** [review]
tag name, otherwise LGTM

File: python/ffi/Cargo.toml (hunk 1 above)
Comment (sanhal) [reply python/ffi/Cargo.toml]: 👍 will go with `genesys-0.1.2`

**Pull request merged**